src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html

```diff
@@ -154,11 +154,11 @@ <h3>
 <ul>
   <li> Other packages can create displayable components for internal
        needs and never make them undisplayable. See
-<a href="https://bugs.java.com/view_bug.do?bug_id=4515058">
+<a href="https://bugs.openjdk.org/browse/JDK-4515058">
 4515058</a>,
-<a href="https://bugs.java.com/view_bug.do?bug_id=4671025">
+<a href="https://bugs.openjdk.org/browse/JDK-4671025">
```

Comment:
And even better it was resolved in 1.5 under this bug id which I guess was still open when this doc was written